packages/common/src/compiler/filtersCompiler.ts

```diff
@@ -485,3 +493,88 @@ export const renderFilterRuleSql = (
         }
     }
 };
+
+/**
+ * Converts a filter rule from custom metrics to dbt meta tags in yml
+ * More info about dbt filters: https://docs.lightdash.com/references/metrics/#available-filter-types
+ */
+const convertFilterOperatorToDbt = (filter: MetricFilterRule): string => {
+    const { values, operator } = filter;
+
+    if (
+        values === undefined ||
+        values.length === 0 ||
+        values[0] === undefined
+    ) {
+        throw new ParameterError(
+            `Filter values are undefined for filter: ${JSON.stringify(filter)}`,
+        );
+    }
+    // Value can be null
+
+    switch (operator) {
+        case FilterOperator.EQUALS:
+        case FilterOperator.NULL:
+            return `${values[0]}`;
```

Comment:
What happens when there are multiple values?
eg:
```
{
    "id": "3c0ca671-26b9-4cd1-b15e-5786740273a3",
    "operator": "equals",
    "values": [
        "1",
        "2",
        "3"
    ],
    "target": {
        "fieldRef": "orders.order_id"
    }
}
```

Reply:
I forgot about that, I'll fix it

Reply:
this should be the right syntax to allow `OR` values  https://docs.lightdash.com/references/metrics/#available-filter-types
![image](https://github.com/user-attachments/assets/d51c3aa4-9269-4485-ab3e-2b9a8e64d712)